config/set/contentrepository-90.php

```diff
@@ -400,6 +401,12 @@
     $rectorConfig->rule(ContextGetCurrentRenderingModeRector::class);
     $rectorConfig->rule(FusionContextCurrentRenderingModeRector::class);
 
+    /**
+     * CreateContentContextTrait
+     */
+    $methodCallToWarningComments[] = new MethodCallToWarningComment(\Neos\Neos\Controller\CreateContentContextTrait::class, 'createContentContext', '!! CreateContentContextTrait::createContentContext() is removed in Neos 9.0.');
```

Comment:
should we tell the user what to do now instead? ... ill gess we need to finetune these notices because just the info that its removed might not be enough?

Reply:
Yes, true. In best case we should provide a link to docs with detailed information. But we don't have the docs yet.